tests/2threads.conf

```diff
@@ -0,0 +1,12 @@
+COMPILE | rrfs | intel | -DAPP=ATM -DCCPP_SUITES=FV3_RAP,FV3_RAP_sfcdiff,FV3_HRRR,FV3_RRFS_v1beta,FV3_RRFS_v1nssl -D32BIT=ON | | fv3 |
```

Comment:
I believe this 2threads.conf file should be deleted.

Reply:
Yes. That was my test conf file for checking that removing an extraneous variable wouldn't break other platforms. I deleted it in hash ea0d11a